model/src/main/kotlin/OrtResult.kt

```diff
@@ -218,8 +218,8 @@ data class OrtResult(
         if (!includeSubProjects) {
             val allSubProjects = mutableSetOf<Identifier>()
 
-            projects.forEach {
-                allSubProjects += dependencyNavigator.collectSubProjects(it)
+            projects.flatMapTo(allSubProjects) {
```

Comment:
Could also be collapsed to `val allSubProjects = projects.flatMapTo(mutableSetOf()) {`.